evaluate/train_image.py

```diff
@@ -1,115 +1,115 @@
                     help='The number of iteration to be averaged over.')
-parser.add_argument('--gpu', '-g', type=int, default=-1, help='GPU to use. Negative value to use CPU')
-parser.add_argument('--cudnn', '-c', action='store_true', help='If this flag is set, cuDNN is enabled.')
+parser.add_argument('--gpu', '-g', type=int, default=-1,
+                    help='GPU to use. Negative value to use CPU')
+parser.add_argument('--cudnn', '-c', action='store_true',
+                    help='If this flag is set, cuDNN is enabled.')
+parser.add_argument('--dry-run', '-d', type=int, default=5,
+                    help='The number of iterations of a dry run '
+                    'not counted towards final timing')
 parser.add_argument('--cache-level', '-C', type=str, default='none',
                     choices=('none', 'memory', 'disk'),
-                    help='This option determines the type of the kernel cache used.'
+                    help='This option determines the type of the kernel '
+                    'cache used.'
                     'By default, memory cache and disk cache are removed '
                     'at the beginning of every iteration. '
                     'Otherwise, elapsed times of each iteration are '
-                    'measured with corresponding cache enabled. '
-                    'If either cache is enabled, this script operates one additional '
-                    'iteration for burn-in before measurement. '
-                    'This iteration is not included in the mean elapsed time.'
-                    'If we do not use GPU, we do not clear cache at all regardless of the value of '
-                    'this option.')
-parser.add_argument('--batchsize', '-b', type=int, default=32, help='Batchsize')
+                    'measured with corresponding cache enabled. ')
+parser.add_argument('--batchsize', '-b', type=int, default=32,
+                    help='Batchsize')
 args = parser.parse_args()
+print(args)
 
 numpy.random.seed(args.seed)
 if args.gpu >= 0:
     cuda.cupy.random.seed(args.seed)
 
 
 in_channels = 3
+label_num = 100
 
 if args.predictor == 'inception-v3':
     predictor = net.inception_v3.InceptionV3(use_cudnn=args.cudnn)
+    model = net.inception_v3.Classifier(predictor)
     in_size = 299
 elif args.predictor == 'alex-owt':
     predictor = net.alex_owt.AlexOWT(use_cudnn=args.cudnn)
+    model = L.Classifier(predictor)
     in_size = 224
 elif args.predictor == 'vgg':
     predictor = net.vgg.VGG(use_cudnn=args.cudnn)
+    model = L.Classifier(predictor)
     in_size = 224
 elif args.predictor == 'resnet-50':
     predictor = net.resnet_50.ResNet50(use_cudnn=args.cudnn)
+    model = L.Classifier(predictor)
     in_size = 224
 else:
     raise ValueError('Invalid architector:{}'.format(args.predictor))
-model = L.Classifier(predictor)
+
 
 if args.gpu >= 0:
     cuda.get_device(args.gpu).use()
     model.to_gpu()
 optimizer = O.SGD()
```

Comment:
The Torch7 deepmark benchmark uses learning rate = 0.0001. Since we are operating on random data anyway, it does not make a practical difference, but might still be good idea to keep the code and model hyperparameters as similar as possible.